### PR TITLE
tests: enable some services in e2e

### PR DESCRIPTION
**Pull request by justinsb**
Starting with the services we definitely need!

### CODE REVIEW

File: dev/tasks/create-test-project [hunk 1]
```diff
@@ -51,3 +51,11 @@ gcloud beta billing projects link --billing-account=${BILLING_ACCOUNT_ID} "${GCP
...
 echo "Setting default project to ${GCP_PROJECT_ID}"
 gcloud config set project "${GCP_PROJECT_ID}"
+
+gcloud services enable \
+  compute.googleapis.com \
+  dataflow.googleapis.com \
+  logging.googleapis.com \
+  monitoring.googleapis.com \
+  redis.googleapis.com \
+  sqladmin.googleapis.com
```
Comment (jasonvigil) [inline dev/tasks/create-test-project:61]: SQL tests depend on `servicenetworking.googleapis.com`

Comment (justinsb) [reply dev/tasks/create-test-project:62]: Added!  I'm sure we'll need lots more over time :-)

Comment (jasonvigil) [reply dev/tasks/create-test-project:62]: Thanks 😄

**Review by jasonvigil — APPROVED** [review]

**Review by yuwenma — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**